mernarezk/assignment2/BinaryTree.java

```diff
@@ -0,0 +1,57 @@
+package Assignment2;
+
+/**
+ * Simple Binary Tree class that takes int values since BOTH problems are
+ * defined by taking in values and for testing simplicity methods params in
+ * assignments take direct int's rather than node references.
+ */
+public class BinaryTree {
+	TreeNode root;
+
+	/**
+	 * For testing and demonstration purposes the isnertion method happens int the

```

Comment:
Please fix word typos: isnertion -> insertion, int the -> in the, imagineable -> imaginable

Reply:
I removed this method and changed insertion to be using inorder + preorder representation of the tree.